x-pack/plugins/monitoring/public/components/shared/toolbar.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiSuperDatePicker, OnRefreshChangeProps } from '@elastic/eui';
+import React, { useContext, useCallback } from 'react';
+import { MonitoringTimeContainer } from '../../application/pages/use_monitoring_time';
+
+export const MonitoringToolbar = () => {
```

Comment:
This component should probably be used in the [page template](https://github.com/elastic/kibana/blob/master/x-pack/plugins/monitoring/public/application/pages/page_template.tsx#L55)